assets/javascripts/modules/confirmation.js

```diff
@@ -0,0 +1,61 @@
+define([
+    '$',
+    'bean',
+    'utils/ajax',
+    'modules/forms/toggleError'
+], function (
+    $,
+    bean,
+    ajax,
+    toggleError
+) {
+    'use strict';
+
+    var $FINISH_ACCOUNT_FORM = $('.js-finish-account');
+    var $FINISH_ACCOUNT_SUBMIT = $('.js-checkout-finish-account-submit');
+    var $FINISH_ACCOUNT_SUCCESS = $('.js-finish-account-success');
+    var $FINISH_ACCOUNT_ERROR = $('.js-finish-account-error');
+    var $FINISH_ACCOUNT_PASSWORD = $('.js-checkout-finish-account-password .js-input');
+    var $FINISH_ACCOUNT_PASSWORD_CONTAINER = $('.js-checkout-finish-account-password');
+    var IS_HIDDEN = 'is-hidden';
+
+    function validatePassword(password) {
+        /**
+         * We have a component which explicitly
+         * does password strength validation.
+         * Why not use that to determine if we shoule submit??
+         * e.g., if zxcvbn says it's not weak?
+         */
+        return password.length >= 6;
```

Comment:
Agree. Maybe we can extend the password module to expose a function that calls zxcvbn and returns a boolean